student.py

```diff
@@ -0,0 +1,8 @@
+class Student:
+    def __init__(self, id, name):
+        self.id = id
+        self.name = name
```

Comment:
Consider adding a 'attendance_date' attribute to the 'Student' class. This would allow tracking of attendance on different dates.